### PR TITLE
add get our data link

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
https://linear.app/electricitymaps/issue/ELE-1553/add-get-our-data-to-link-above-charts-improve-spacing
## Description
Adds a new link with improved spacing as a subtitle to the charts. Also doesn't render the price chart title when there is no price chart.

### CODE REVIEW

File: web/src/features/charts/ChartTitle.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { useAtom } from 'jotai';
 import { useTranslation } from 'translation/translation';
 import { formatTimeRange } from 'utils/formatting';
 import { timeAverageAtom } from 'utils/state/atoms';
+import { HiOutlineArrowDownTray } from 'react-icons/hi2';
```
Comment (madsnedergaard) [inline web/src/features/charts/ChartTitle.tsx:5]: Haha, love the literal name here 😄

File: web/src/features/charts/ChartTitle.tsx [hunk 2]
```diff
@@ -21,13 +22,26 @@ export function ChartTitle({ translationKey }: Props) {
...
+    <>
+      <h3 className="pt-3 pb-1 text-md">
+        {localExists
+          ? __(`${translationKey}.${timeAverage}`)
+          : __(
+              `${translationKey}.default`,
+              formatTimeRange(localDefaultExists ? i18n.language : 'en', timeAverage)
+            )}
+      </h3>
+      <div className="message flex flex-row items-center pb-2 text-xs no-underline">
```
Comment (madsnedergaard) [inline web/src/features/charts/ChartTitle.tsx:34]: The text size feels very small and hard to read - it's 9.6px now vs 10.6px in the old app. Maybe we can consider renaming it to something shorter and then bump text size? :D

Comment (madsnedergaard) [inline web/src/features/charts/ChartTitle.tsx:34]: Hmm, it's a bit of a Tailwind antipattern I think that we at some point introduced the `.message` class with styling - it's probably not for now, but we should redo this at some point :)

For now though, would you mind making the link-color a tad darker here? I think it's a bit too bright for proper contrast:
Screenshot 2023-01-06 at 16 37 39
Screenshot 2023-01-06 at 16 38 32

File: web/src/features/charts/PriceChart.tsx [hunk 3]
```diff
@@ -22,6 +21,9 @@ function PriceChart({ datetimes, timeAverage }: PriceChartProps) {
   const { chartData, layerFill, layerKeys, layerStroke, valueAxisLabel, markerFill } =
     data;
 
+  if (!chartData[24].layerData.price) {
+    return null;
+  }
```
Comment (madsnedergaard) [inline web/src/features/charts/PriceChart.tsx:26]: Seems like it's only 24 on hourly, right?
Switching to yearly and opening the panel breaks the app :)

**Review by madsnedergaard — COMMENTED** [review]
Thanks for fixing this 🙏 A few minor comments below.

Also as a general thing, it would be really nice if you could include screenshots for PRs with simple visual changes, so the reviewer doesn't necessarily need to switch branch and run the app when reviewing the PR 😄
As a longer term solution it would be really cool do to automated preview builds (like Netlify and [Vercel](https://vercel.com/docs/concepts/deployments/preview-deployments) does) to make it easier to test things!

File: web/src/features/charts/ChartTitle.tsx [hunk 4]
```diff
@@ -21,13 +22,26 @@ export function ChartTitle({ translationKey }: Props) {
...
+          ? __(`${translationKey}.${timeAverage}`)
+          : __(
+              `${translationKey}.default`,
+              formatTimeRange(localDefaultExists ? i18n.language : 'en', timeAverage)
+            )}
+      </h3>
+      <div className=" flex flex-row items-center pb-2 text-center text-sm  ">
+        <HiOutlineArrowDownTray className="min-w-[12px]" size={12} />
+        <a
+          href="https://electricitymaps.com/?utm_source=app.electricitymap.org&utm_medium=referral&utm_campaign=country_panel"
```
Comment (VIKTORVAV99) [inline web/src/features/charts/ChartTitle.tsx:37]:
```suggestion
          href="https://electricitymaps.com/?utm_source=app.electricitymaps.com&utm_medium=referral&utm_campaign=country_panel"
```
Shouldn't we use the new domain?

File: web/src/features/charts/PriceChart.tsx (hunk 3 above)
Comment (tonypls) [reply web/src/features/charts/PriceChart.tsx]: Updated :D

**Review by madsnedergaard — APPROVED** [review]

**Pull request merged**